.circleci/config.yml

```diff
@@ -119,263 +239,70 @@ jobs:
-          aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
-    working_directory: /app
-    steps:
-      - run: yarn bundle-report:client --json webpack-stats.json
-      - run: npx relative-ci-agent
-
-not_main_or_staging_or_release: &not_main_or_staging_or_release
-  filters:
-    branches:
-      ignore:
-        - main
-        - staging
-        - release
-
-not_staging_or_release: &not_staging_or_release
-  filters:
-    branches:
-      ignore:
-        - staging
-        - release
-
-only_main: &only_main
-  context:
-    - hokusai
-    - slack-orb
-  filters:
-    branches:
-      only: main
-
-only_release: &only_release
-  context:
-    - hokusai
-    - slack-orb
-  filters:
-    branches:
-      only: release
-
 workflows:
   default:
     jobs:
       - detect-secrets:
           <<: *not_main_or_staging_or_release
 
-      # Main build
-      - artsy-remote-docker/buildkit-build:
-          <<: *not_staging_or_release
-          context: hokusai
-          executor: hokusai/beta
-          name: builder-image-build
-          pre-steps:
-            - run:
-                command: echo 'export BUILD_TARGET="builder";' >> $BASH_ENV
-
-      - artsy-remote-docker/buildkit-push:
-          <<: *not_staging_or_release
-          context: hokusai
-          executor: hokusai/beta
-          name: builder-image-push
-          requires:
-            - builder-image-build
-          pre-steps:
-            - run:
-                command: echo 'export BUILD_TARGET="builder";' >> $BASH_ENV
+      # Danger
+      - node/run:
+          <<: *not_main_or_staging_or_release
+          context:
+            - danger-github-oss
+          name: check-pr
+          pkg-manager: yarn
+          yarn-run: danger:ci
+          version: 18.15.0
 
-      # Electron Runner build
-      - artsy-remote-docker/buildkit-build:
-          <<: *not_staging_or_release
-          context: hokusai
-          executor: hokusai/beta
-          name: electron-runner-image-build
-          requires:
-            - builder-image-build
-          pre-steps:
-            - run:
-                command: echo 'export BUILD_TARGET="electron-runner";' >> $BASH_ENV
+      # Relay
+      - node/run:
+          <<: *not_main_or_staging_or_release
+          name: relay
+          pkg-manager: yarn
+          yarn-run: relay
+          version: 18.15.0
 
-      - artsy-remote-docker/buildkit-push:
+      # Type-check
+      - node/run:
           <<: *not_staging_or_release
-          context: hokusai
-          executor: hokusai/beta
-          name: electron-runner-image-push
-          requires:
-            - electron-runner-image-build
-          pre-steps:
-            - run:
-                command: echo 'export BUILD_TARGET="electron-runner";' >> $BASH_ENV
+          name: type-check
+          pkg-manager: yarn
+          yarn-run: type-check
+          version: 18.15.0
 
-      # Test steps
+      # Jest Tests
       - test:
```

Comment:
It seems test results (and artifacts) failed to upload so we'll lose test insights. The paths may need to update after migrating to use CircleCI images.

![Screenshot 2024-09-24 at 9 05 23 AM](https://github.com/user-attachments/assets/5f8dbc33-44f9-43c5-a615-0c2ef667550c)